docs/programming/arduino.md

```diff
@@ -43,77 +44,76 @@ performing an action with that pin. You can read about the possible pin
 
 ``` python
-robot.arduino.pins[4].mode = GPIOPinMode.DIGITAL_INPUT
+robot.arduino.pins[4].mode = GPIOPinMode.INPUT
 
-pin_value = robot.arduino.pins[4].digital_state
+pin_value = robot.arduino.pins[4].digital_value
 ```
 
-### `GPIOPinMode.DIGITAL_INPUT_PULLUP`
-
-Same as `GPIOPinMode.DIGITAL_INPUT`, but with an internal [pull-up
-resistor](https://learn.sparkfun.com/tutorials/pull-up-resistors)
-enabled.
+Some external switches may require a pull up resistor,
+`GPIOPinMode.INPUT_PULLUP`, is the same as INPUT but also enables an internal [pull-up
+resistor](https://learn.sparkfun.com/tutorials/pull-up-resistors).
 
 ``` python
-robot.arduino.pins[4].mode = GPIOPinMode.DIGITAL_INPUT_PULLUP
+robot.arduino.pins[4].mode = GPIOPinMode.INPUT_PULLUP
 
-pin_value = robot.arduino.pins[4].digital_state
+pin_value = robot.arduino.pins[4].digital_value
 ```
 
-### `GPIOPinMode.DIGITAL_OUTPUT`
+### Digital Output
 
-In this mode, we can set binary values of `0V` or `5V` to the pin.
+Digital outputs can be used to set binary values of `0V` or `5V` to the pin.
+This can be used to turn an LED on and off for example.
 
 ``` python
-robot.arduino.pins[4].mode = GPIOPinMode.DIGITAL_OUTPUT
-robot.arduino.pins[6].mode = GPIOPinMode.DIGITAL_OUTPUT
+robot.arduino.pins[4].mode = GPIOPinMode.OUTPUT
+robot.arduino.pins[6].mode = GPIOPinMode.OUTPUT
 
-robot.arduino.pins[4].digital_state = True
-robot.arduino.pins[6].digital_state = False
+robot.arduino.pins[4].digital_value = True
+robot.arduino.pins[6].digital_value = False
 ```
 
-### `GPIOPinMode.ANALOGUE_INPUT`
+### Analog Input
 
-Certain sensors output analogue signals rather than digital ones, and so
-have to be read differently. The Arduino has six analogue inputs, which
-are labelled `A0` to `A5`; however pins `A4` and `A5` are reserved and
-cannot be used.
+Certain sensors output analog signals rather than digital ones, and so
+have to be read differently. The Arduino has six analog inputs, which
+are labelled `A0` to `A5`.
 
 !!! tip
-    Analogue signals can have any voltage, while digital signals can only
-    take on one of two voltages. You can read more about digital vs analogue
+    Analog signals can have any voltage, while digital signals can only
+    take on one of two voltages. You can read more about digital vs analog
     signals [here](https://learn.sparkfun.com/tutorials/analog-vs-digital).
 
 ``` python
-from sbot import AnaloguePin
+from sbot import AnalogPins
 
-robot.arduino.pins[AnaloguePin.A0].mode = GPIOPinMode.ANALOGUE_INPUT
+robot.arduino.pins[AnalogPins.A0].mode = GPIOPinMode.INPUT
 
-pin_value = robot.arduino.pins[AnaloguePin.A0].analogue_value
+pin_value = robot.arduino.pins[AnalogPins.A0].analog_value
 ```
 
 !!! tip
-    The values are the voltages read on the pins, between 0 and 5.
+    The values are the voltages read on the pin, between 0 and 5.
 
-## Ultrasound Sensors
+### Ultrasound Sensors
 
 You can also measure distance using an ultrasound sensor from the arduino.
```

Comment:
Suggestion: We should explicitly note here that the return unit is millimetres.

Reply:
Fixed in [cb70f69](https://github.com/sourcebots/docs/pull/217/commits/cb70f6955ee0b2bed3180f38138832923eeb4ed9)